### PR TITLE
bpo-12822: use monotonic clock for condvar if possible

### PR DESCRIPTION
**Pull request by methane**
https://bugs.python.org/issue12822

### CODE REVIEW

File: Makefile.pre.in [hunk 1]
```diff
@@ -939,7 +939,8 @@ regen-opcode-targets:
 		$(srcdir)/Python/opcode_targets.h.new
 	$(UPDATE_FILE) $(srcdir)/Python/opcode_targets.h $(srcdir)/Python/opcode_targets.h.new
 
-Python/ceval.o: $(srcdir)/Python/opcode_targets.h $(srcdir)/Python/ceval_gil.h
+Python/ceval.o: $(srcdir)/Python/opcode_targets.h $(srcdir)/Python/ceval_gil.h \
+		$(srcdir)/Python/condvar.h
 
 Python/frozen.o: $(srcdir)/Python/importlib.h $(srcdir)/Python/importlib_external.h \
 		$(srcdir)/Python/importlib_zipimport.h
```
Comment (eamanu) [inline Makefile.pre.in:946]: Is not part of the your PR, but I just see. Could you add a whitespace at the start of the line, please? :-)

**Review by eamanu — COMMENTED** [review]
lgtm but for some reason on mac the tests fail

File: Makefile.pre.in (hunk 1 above)
Comment (methane) [reply Makefile.pre.in:946]: I can't get what do you mean.
This is Makfefile, not Python.  So I use TAB instead of SPACE.

Comment (methane) [reply Makefile.pre.in:946]: When wrapping dependency files, two TABs are used.
We don't align vertically.  For example:

image

**Pull request merged**